api/handlers/admin.go

```diff
@@ -0,0 +1,88 @@
+package handlers
+
+import (
+	"blogs/api/services"
+	"blogs/common/constants"
+	"blogs/common/dto"
+	"blogs/pkg/loggers"
+	"net/http"
+	"strconv"
+
+	"github.com/labstack/echo/v4"
+)
+
+type AdminHandler struct {
+	services.AdminServices
+}
+
+// retrieve every users records
+func (handler *AdminHandler) GetUsers(ctx echo.Context) error {
+	var limit, offset int
+
+	limit_param := ctx.QueryParam("limit")
+	if limit_param == "" {
+		limit = constants.Default_Limit
+	} else {
+		convLimit, err := strconv.Atoi(limit_param)
+		if err != nil {
+			loggers.Warn.Println(err)
+			return ctx.JSON(http.StatusBadRequest, dto.ResponseJson{
+				Error: err.Error(),
+			})
+		}
+		limit = convLimit
+	}
+
+	offset_param := ctx.QueryParam("offset")
+	if offset_param == "" {
+		offset = constants.Default_Offset
+	} else {
+		convOffset, err := strconv.Atoi(offset_param)
+		if err != nil {
+			loggers.Warn.Println(err)
+			return ctx.JSON(http.StatusBadRequest, dto.ResponseJson{
+				Error: err.Error(),
+			})
+		}
+		offset = convOffset
+	}
+
+	getRole := (ctx.Get("role"))
```

Comment:
* dont prefix variable names with get, create, update etc....
* u could use roleCtx instead
* make this change whereever necessary
```suggestion
	getRole := (ctx.Get("role"))
    roleCtx := (ctx.Get("role"))
```